src/main/kotlin/io/github/mojira/arisa/modules/KeepPrivateModule.kt

```diff
@@ -23,12 +23,12 @@ class KeepPrivateModule(
             setPrivate()
 
             val markedTime = comments.first(::isKeepPrivateTag).created
-            val changedTime = changeLog.lastOrNull(::isSecurityChange)?.created
             val securityChange = changeLog
-                .lastOrNull() { isSecurityChangeToPublic(it, securityLevel, project.privateSecurity) }
+                .lastOrNull() { isSecurityChangeToPublic(it, project.privateSecurity) }
+            val changedTime = securityChange?.created
```

Comment:
As proposed in #609, uses `securityChange` to determine `changedTime` for consistency.